source/js/components/join/join.jsx

```diff
@@ -2,7 +2,9 @@ import React from "react";
 import ReactGA from "react-ga";
 import ReactDOM from "react-dom";
 import classNames from "classnames";
-import basketSignup from "../../basket-signup.js";
+import CountrySelect from "../petition/country-select.jsx";
+import get from "../petition/locales";
```

Comment:
This is too generic a name, let's change this to something that makes its role clearer in the code:
```suggestion
import getText from "../petition/locales";
```
(we can do this because `petition/locales.js` exports `get` as a default export, which means importing code is free to call it whatever it wants. It simply "gets something" as import result, and gets to store that by whatever name makes sense for the importer)

Reply:
> we can do this because petition/locales.js exports get as a default export, which means importing code is free to call it whatever it wants.

Oooo that's really good to know.